doc/cabal-commands.rst

```diff
@@ -131,6 +136,9 @@ cause extra store packages to be built (for example,
 ``--enable-profiling`` will automatically make sure profiling libraries
 for all transitive dependencies are built and installed.)
 
+When building a script, the executable is cached under the cabal directory.
```

Comment:
Is it also cached when executed only via `v2-run`?

Reply:
yes

Reply:
Please document that, thank you :)